src/transaction.rs

```diff
@@ -1,6 +1,9 @@
 //! Implementation for setting up, signing, estimating gas and executing
 //! transactions on the Ethereum network.
 
+#![allow(clippy::large_enum_variant)]
```

Comment:
Is it possible to allow this only for the enum/type that large/complex?

Reply:
We address this in a later commit, I just wanted this to get through the CI for now.